### DISCUSSION TITLE
How to resolve the tag name conflicts in Stencil v4?

### DISCUSSION THREAD

**Discussion by stout-ni** [stenciljs/core#5936] [Stencil Testing]
I noticed that tagNameTransform doesn't work in Stencil 4.x, so what would be an alternative solution to solve the tag name conflicts?

**Comment by christian-bromann** [comment]
Hey @stout-ni 👋 

I don't think there are alternatives when a certain feature breaks. I came across this when working on #5853 and was about to remove it when I realised that this is used a lot within the ecosystem. Can you do me a favor an raise an issue for this?
